tests/test_constraints_polars.py

```diff
@@ -0,0 +1,111 @@
+"""Test Polars' implementations of constraints."""
```

Comment:
It's not their implementation 🙃 
```suggestion
"""Test Polars implementations of constraints."""
```

Reply:
changed